spec/spec_helper_acceptance.rb

```diff
@@ -1,45 +1,35 @@
-require 'beaker-rspec'
-require 'beaker-puppet'
-require 'beaker/puppet_install_helper'
-require 'beaker/module_install_helper'
+require 'voxpupuli/acceptance/spec_helper'
 
-run_puppet_install_helper unless ENV['BEAKER_provision'] == 'no'
-
-RSpec.configure do |c|
+configure_beaker do |host|
   # Configure all nodes in nodeset
-  c.before :suite do
-    install_module
-    install_module_dependencies
-
-    install_server_packages = %(
-      if $facts['os']['name'] == 'CentOS' {
-        package { 'epel-release':
-          ensure => present,
-        }
-      }
-
-      $package_name = $facts['os']['family'] ? {
-        'Debian' => 'netcat-openbsd',
-        'RedHat' => 'nc',
-        default  => 'netcat',
-      }
-      package { $package_name:
+  install_server_packages = %(
+    if $facts['os']['name'] == 'CentOS' {
+      package { 'epel-release':
         ensure => present,
       }
-    )
-    apply_manifest_on(hosts_as('vpnserver'), install_server_packages, catch_failures: true)
+    }
 
-    install_client_packages = %(
-      if $facts['os']['name'] == 'CentOS' {
-        package { 'epel-release':
-          ensure => present,
-        }
-      }
+    $package_name = $facts['os']['family'] ? {
+      'Debian' => 'netcat-openbsd',
+      'RedHat' => 'nc',
+      default  => 'netcat',
+    }
+    package { $package_name:
+      ensure => present,
+    }
+  )
+  apply_manifest_on(hosts_as('vpnserver'), install_server_packages, catch_failures: true)
```

Comment:
This is not correct. It runs it for every host, rather than just once on a selected set of nodes. This is because `configure_beaker` iterates over every host. I haven't thought about a clean way to do this.